lib/features/public_asset/presentation/public_asset_controller.dart

```diff
@@ -114,7 +115,28 @@ class PublicAssetController extends BaseController {
   void handleFailureViewState(Failure failure) {
     super.handleFailureViewState(failure);
     if (failure is CreatePublicAssetFailureState) {
-      isUploading.value = false;
+      _handleCreatePublicAssetFailureState();
+    } else if (failure is PublicAssetOverQuotaFailureState) {
+      _handlePublicAssetOverQuotaFailureState(failure);
+    }
+  }
+
+  void _handlePublicAssetOverQuotaFailureState(PublicAssetOverQuotaFailureState failure) {
+    isUploading.value = false;
+    if (currentOverlayContext != null && currentContext != null) {
+      appToast.showToastErrorMessage(
+        currentOverlayContext!,
+        failure.publicAssetQuotaExceededException.message
+          ?? AppLocalizations.of(currentContext!).unknownError);
+    }
+  }
+
+  void _handleCreatePublicAssetFailureState() {
+    isUploading.value = false;
+    if (currentOverlayContext != null && currentContext != null) {
+      appToast.showToastErrorMessage(
+        currentOverlayContext!,
+        AppLocalizations.of(currentContext!).unknownError);
```

Comment:
should be a dedicated message to image of signature.

Reply:
Done